CoroutineTemplate/README.md

```diff
@@ -3,23 +3,38 @@
 
 ### Setup
 - Clone the project
-- Checkout our main development branch `kotlin`
 - Run the project with Android Studio
 
 ### Linter and static code analysis
 
-1. Check Style:
+- Lint:
 
 ```
-$ ./gradlew checkStyle
+$ ./gradlew lint
 ```
 
-Report is located at: `./app/build/reports/checkstyle/`
+Report is located at: `./app/build/reports/lint/`
 
-2. Detekt
+- Detekt
 
 ```
 $ ./gradlew detekt
 ```
 
-Report is located at: `./build/reports/detekt.html`
+Report is located at: `./build/reports/detekt`
+
+### Testing
+
+- Run unit testing:
+
+```
+$ ./gradlew test
```

Comment:
When I run `./gradlew jacocoTestReport` it works correctly, but once I run `./gradlew test` it fails:
<img width="1263" alt="Screen Shot 2565-04-07 at 10 35 05" src="https://user-images.githubusercontent.com/18277915/162115100-9244f74f-23d3-4e6a-9e1c-2bbe4c338816.png">

Do you have the same? 💭

Reply:
@Tuubz As I explained here https://github.com/nimblehq/android-templates/pull/139/files/84cf42f7aa3d8287f9119742ce3d921b31e69111#r844832759, we need to keep `EmptyHiltActivity` in `debug` flavor but avoid to have it in `release` flavor so the test can not be executed in `release` flavor.

I think I will need to scope this command to only on `debug` flavor. Is that good? 🤓

Reply:
Not being able to run tests on `release` flavor does sound a bit concerning

Anyways, it's still an improvement from before

Shall we add a TODO for now? In case we can apply it for `release` to in the future 🙏

Reply:
@Tuubz Actually we still can config to make the test work on `release` flavor but it's not in this PR scope. Let me add some comments on `EmptyHiltActivity` class 🙋

Reply:
@Tuubz Mind changed, I don't think unit testing on release build is a thing we normally do. I think we should keep the current config as it is without TODO comment.

Reply:
**Conclusion**: Approve this issue for now and create an issue soon to go to the voting process

Reply:
**Conclusion**: Approve this issue for now and create an issue soon to go to the voting process